### ISSUE TITLE
Bloomtown: A Different Story

### ISSUE THREAD

**Issue by MESHuDB** [p1xel8ted/UltrawideFixes#44]
The game runs on the Unity engine. I have tried repeatedly to add BepInEx, but unfortunately the game is Stripped. Although I know the solution, it did not work for me. Can you add ultrawide fix ?

**Comment by p1xel8ted** [comment]
You can bypass BepInEx altogether and inject a fix dll using Doorstop directly, however the game spits out exceptions left right and center, so it's not looking good.

**Comment by MESHuDB** [comment]
Well already fixed
😅
But I have no idea about adding ultrawide fix mod. Do you have any idea about how it works or can you do it yourself?

**Comment by p1xel8ted** [comment]
No, there is no fix using injection at this stage, because as I said, the game throws exceptions (errors) and bugs out.
